### DISCUSSION TITLE
NPM in two networks dockers & real lan

### DISCUSSION THREAD

**Discussion by djtomybcn** [NginxProxyManager/nginx-proxy-manager#3707] [General]
Hi!

I've NPM running in a docker with a macvlan with a "real ip" from my lan like 192.168.0.xx and serve some services like Homeassitant (192.168.0.xx) that are in another machine (raspberrypi) that have an ip from my network. This woks like a charm and I can access from inside and outside using the domain name.

So as not to have all my dockers in the home lan I want to setup a "backend network" with a range 10.30.0.0 (for example)

It's possible to do that, or nginx only works on the same lan?

I've tried to configure nginx with two networks with dockers and works the redirections to services that are in "real lan" and "backend lan" when I try from a device that are in the 192.168.0.xx lan. But when I try to enter from internet, 4g mobile for example, it's not working.

Any ideas?

**Comment by nckrwlmn** [comment]
Make sure your domain name is pointing to the IP address of your router and that it is forwarding port 80 and 443 to your NPM server.

You should be able to proxy to anywhere as long as the NPM has a route to that anywhere.
